workflow/energy_rating_index.rb

```diff
@@ -153,57 +151,81 @@ def read_output(design, designdir, output_hpxml_path)
   map_tsv_data = CSV.read(File.join(designdir, "map_hvac.tsv"), headers: false, col_sep: "\t")
   design_output[:elecHeatingBySystem] = {}
   design_output[:fuelHeatingBySystem] = {}
+  design_output[:elecHeatingBySystemDSE] = {}
+  design_output[:fuelHeatingBySystemDSE] = {}
   design_output[:loadHeatingBySystem] = {}
   design_output[:hpxml_heat_sys_ids].each do |sys_id|
     ep_output_names = get_ep_output_names_for_hvac_heating(map_tsv_data, sys_id, hpxml_doc, design)
     keys = "'" + ep_output_names.map(&:upcase).join("','") + "'"
+
     # Electricity Use
     vars = "'" + get_all_var_keys(OutputVars.SpaceHeatingElectricity).join("','") + "'"
     query = "SELECT SUM(ABS(VariableValue)/1000000000) FROM ReportVariableData WHERE ReportVariableDataDictionaryIndex IN (SELECT ReportVariableDataDictionaryIndex FROM ReportVariableDataDictionary WHERE VariableType='Sum' AND KeyValue IN (#{keys}) AND VariableName IN (#{vars}) AND ReportingFrequency='Run Period' AND VariableUnits='J')"
     design_output[:elecHeatingBySystem][sys_id] = get_sql_query_result(sqlFile, query)
+
     # Fuel Use
     vars = "'" + get_all_var_keys(OutputVars.SpaceHeatingFuel).join("','") + "'"
     query = "SELECT SUM(ABS(VariableValue)/1000000000) FROM ReportVariableData WHERE ReportVariableDataDictionaryIndex IN (SELECT ReportVariableDataDictionaryIndex FROM ReportVariableDataDictionary WHERE VariableType='Sum' AND KeyValue IN (#{keys}) AND VariableName IN (#{vars}) AND ReportingFrequency='Run Period' AND VariableUnits='J')"
     design_output[:fuelHeatingBySystem][sys_id] = get_sql_query_result(sqlFile, query)
+
     # Disaggregated Fan Energy Use
     ems_keys = "'" + ep_output_names.select { |name| name.include? "Heating" }.join("','") + "'"
     query = "SELECT SUM(ABS(VariableValue)/1000000000) FROM ReportVariableData WHERE ReportVariableDataDictionaryIndex IN (SELECT ReportVariableDataDictionaryIndex FROM ReportVariableDataDictionary WHERE VariableType='Sum' AND KeyValue='EMS' AND VariableName IN (#{ems_keys}) AND ReportingFrequency='Run Period' AND VariableUnits='J')"
     fan_pump_output = get_sql_query_result(sqlFile, query)
     design_output[:elecHeatingBySystem][sys_id] += fan_pump_output
+
+    # apply dse to scale up energy use excluding no distribution systems
+    if design_output[:hpxml_dse_heats].nil?
+      design_output[:fuelHeatingBySystemDSE][sys_id] = design_output[:fuelHeatingBySystem][sys_id]
+      design_output[:elecHeatingBySystemDSE][sys_id] = design_output[:elecHeatingBySystem][sys_id]
+    else
+      design_output[:fuelHeatingBySystemDSE][sys_id] = design_output[:fuelHeatingBySystem][sys_id] / design_output[:hpxml_dse_heats][sys_id]
+      design_output[:elecHeatingBySystemDSE][sys_id] = design_output[:elecHeatingBySystem][sys_id] / design_output[:hpxml_dse_heats][sys_id]
```

Comment:
Is there a need to have both DSE-unadjusted and DSE-adjusted consumption values? If not, it seems simpler to just update the existing, e.g., `design_output[:fuelHeatingBySystem]` values in place so that you don't have to change so many references. It would also prevent someone from accidentally referencing the DSE-unadjusted values.

Reply:
Yes I did it in that way at the beginning, there's a total energy check which prevents postprocessing. [Right here](https://github.com/NREL/OpenStudio-ERI/blob/master/workflow/energy_rating_index.rb#L329-L345)

Reply:
Ah, right. In that case, how about removing the unadjusted keys from the hash after those checks? That way they aren’t accidentally referenced. 

And how about using FooSystemRaw (or equivalent) and FooSystem, instead of FooSystem and FooSystemDSE?

Reply:
That sounds like a good idea!